### ISSUE TITLE
anyOf is not matched requirements

### ISSUE THREAD

**Issue by dawnbreaks** [VerbalExpressions/JavaVerbalExpressions#59]
How can JavaVerbalExpressions build a simple regex like this:   [\d,]+

``java
VerbalExpression.regex().anyOf("0123456789,").oneOrMore().build()
``
It 's too ugly! How about this one: [a-k\s\d,]+   ?

**Comment by lanwen** [comment]
Can't see why your variant better than original

**Comment by dawnbreaks** [comment]
@lanwen It 's ridiculous to list all the Word char and all digit char in the parameter in the anyOf method.

**Comment by lanwen** [comment]
lib already has `.wordChar().nonWordChar().space().nonSpace().digit().nonDigit()`

**Comment by dawnbreaks** [comment]
No. Please tell me how to use verbalExpressions to represent this regex:  [a-k\s\d,]+

**Comment by lanwen** [comment]
`regex().add("[").wordChar().space().digit().add("]").oneOrMore()`

`regex().anyOf("\\w\\s\\d")`

`regex().capture().digit().or("\\w").or("\\s").endCapture().oneOrMore()`

`regex().add("[a-k\s\d,]+")`

You can use raw regex if it looks more compact

**Comment by dawnbreaks** [comment]
I think java regex is more readable than verbal expressions.

**Comment by lanwen** [comment]
In some cases it's true! VE is mostly for those who don't know how to type simple regex. Or where you should write something that should be human-readable (for example in tests).

**Comment by eduardohmg** [comment]
It's some late, but I think with a indentation it can be better:

```
.add("[")
.wordChar().space().digit()
.add("]")
.oneOrMore()
```

You are right, sometimes regex is better than ve, principally for those who know regex enough. But we have a lot of thing to improve our code, whatever we use.
